src/lib/snark_worker_lib/intf.ml

```diff
@@ -5,20 +5,16 @@ open Signature_lib
 let command_name = "snark-worker"
 
 module type Inputs_intf = sig
-  module Sparse_ledger : sig
-    type t [@@deriving bin_io, sexp]
-  end
-
   module Transaction_witness : sig
-    type t [@@deriving bin_io, sexp]
+    type t [@@deriving bin_io, sexp, version]
   end
 
   module Transaction : sig
-    type t [@@deriving bin_io, sexp]
+    type t [@@deriving bin_io, sexp, version]
   end
 
   module Proof : sig
-    type t [@@deriving bin_io, sexp]
+    type t [@@deriving bin_io, sexp, version]
```

Comment:
There's a mismatch between how `Statement` is treated, and `Proof`, `Transaction`, etc. here.

For `Statement`, we have the `Stable` hierarchy show, but not for the others. Can/should they be consistent?